### PR TITLE
Merge branch 'master' of https://github.com/Sigurd-Hukou/DX-OI-source…

### PR DESCRIPTION
**Pull request by Colara**
…-code
FT Summary add Item: Sockets and Fit for Socket Counter 

Signed-off-by: Colara <cola_shen@Sigurd.com.tw>

### CODE REVIEW

**Comment by Colara** [comment]
FT Summary add Item: Sockets and Fit for Socket Counter

**Comment by Colara** [comment]
FT Summary add Item: Sockets and Fit for Socket Counter

**Comment by Sigurd-JiaHui** [comment]
推撥測試

File: csicDXAuto_Sigurd_CUS.java [hunk 1]
```diff
@@ -1067,6 +1067,16 @@ static public void generate_userFlag_file() {	//20140704 by Chia_Hui
 			if(testTypeStr.equalsIgnoreCase("Final")){	//20171121
 				barcode_OI_content += "SG_Summary_Naming=" + summaryfileStr_MES.substring(0, summaryfileStr_MES.length()-4) + "\n";
 				barcode_OI_content += "SG_Summary_LOT_START_TIME=" + Summary_LOT_START_TIME + "\n";	//20171226
+				
+				barcode_OI_content += "SG_KIT_Number=" + kit_No_panel.getText() + "\n";	//20171229-----Start
+				for(int x=0; x < barcode_duts_int;x++){	
+					if (!SocketCount[x].equals("")){		            		
+//						tmpStr += "ProductionSetUserValue " + "site" + x + ": " + SocketCount[x] + "\n";
```
Comment (Sigurd-JiaHui) [inline csicDXAuto_Sigurd_CUS.java]: 一些沒用到的註解以後就順手拿掉吧~

Comment (Sigurd-JiaHui) [reply csicDXAuto_Sigurd_CUS.java]: 測試推撥功能

**Comment by Colara** [comment]
This is old version

**Pull request closed without merging**